packages/slider/styles.css

```diff
@@ -0,0 +1,113 @@
+:root {
+  --reach-slider: 1;
+  --highlight-color: #1159a6;
```

Comment:
Should we prefix all our CSS vars with `--reach-*`? I'm just thinking about people overriding this stuff in the future. Honestly not sure the best strategy to use.

Reply:
Went ahead and removed this variable altogether. We aren't using CSS vars anywhere else so no need to start now.